.github/workflows/build-test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             fail-fast: false
```

Comment:
We should also add new build to jenkinsfile, shouldn't we?

Reply:
Right, fixed